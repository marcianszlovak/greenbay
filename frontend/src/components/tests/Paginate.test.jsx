import React from 'react';
import renderer from 'react-test-renderer';

import Paginate from '../Paginate';
import { MemoryRouter } from 'react-router-dom';

it('should render correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Paginate isAdmin={false} keyword={''} page={3} pages={6} />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
