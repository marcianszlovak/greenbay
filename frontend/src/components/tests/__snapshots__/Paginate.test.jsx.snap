// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<ul
  className="pagination"
>
  <li
    className="page-item"
  >
    <a
      className="page-link"
      href="/page/1"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      1
    </a>
  </li>
  <li
    className="page-item"
  >
    <a
      className="page-link"
      href="/page/2"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      2
    </a>
  </li>
  <li
    className="page-item active"
  >
    <span
      className="page-link"
      disabled={false}
      href="/page/3"
      onClick={[Function]}
    >
      3
      <span
        className="sr-only"
      >
        (current)
      </span>
    </span>
  </li>
  <li
    className="page-item"
  >
    <a
      className="page-link"
      href="/page/4"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      4
    </a>
  </li>
  <li
    className="page-item"
  >
    <a
      className="page-link"
      href="/page/5"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      5
    </a>
  </li>
  <li
    className="page-item"
  >
    <a
      className="page-link"
      href="/page/6"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      6
    </a>
  </li>
</ul>
`;
